packages/blank-theme/src/components/common/icon/index.tsx

```diff
@@ -0,0 +1,61 @@
+"use client"
+import { Icon, IconProps } from "@iconify/react"
+import { Box, BoxProps } from "@mui/material"
+type CIconProps = {
+   color?: string
+   hoverColor?: string
+   size?: number
+   icon: string
+   sx?: BoxProps["sx"]
+   iconProps?: IconProps
+} & BoxProps
+export default function CIcon({ color, hoverColor, size = 24, icon, sx, iconProps, ...props }: CIconProps) {
+   return (
+      <Box
+         sx={{
+            color: color ? color : (theme) => theme.palette.text.secondary,
+            fontSize: size,
+            "&:hover": {
+               color: hoverColor
+            },
+            p: 0,
+            m: 0,
+            component: "span",
+            justifyContent: "center",
+            alignItems: "center",
+            display: "flex",
+            ...sx
+         }}
+         {...props}>
+         <Icon icon={icon} {...iconProps} />
+      </Box>
+   )
+}
+
+import type { SVGProps } from "react"
+
+export function SpinnersClock(props: SVGProps<SVGSVGElement>) {
+   return (
+      <svg xmlns='http://www.w3.org/2000/svg' width='1em' height='1em' viewBox='0 0 24 24' {...props}>
```

Comment:
Is it okay to directly use svg . other sv might too long on code.

Reply:
I don't recommend using the `SVG` code icon. All this SVG code was added by the new worker. I recommend using the `Iconify` library for any icons.

Reply:
well, should included in coding guideline

Reply:
@zelal-dev vai, yes, you are right. I should guide him, and I was doing so, and told him to use the `iconify` icon library.